kong/plugins/acme/api.lua

```diff
@@ -71,4 +71,50 @@ return {
       return kong.response.exit(202, { message = "Renewal process started successfully" })
     end,
   },
+  ['/acme-domains/:host'] = {
+    DELETE = function(self)
+      local host = self.params.host
+      -- retrieve acme_domain for given host
+      local entity, err = kong.db.acme_domain:select_by_name(host)
+      if err then
```

Comment:
doesn't `not found` cause a an error ?

Reply:
no, the `err` here is db or some other unexpected error.

i.e , if a record is not found, we still get `err` as nil